cualid/label.py

```diff
@@ -26,7 +26,7 @@ def get_barcodes(input,
                  x_start=1.9,
                  y_start=257.2):
 
-    ids = [e.strip().split('\t')[1] for e in input]
+    ids = [e.strip().split('\t')[0] for e in input]
```

Comment:
It originally checked the second column where the cual-id was, but I
changed it to read the first. I think you may have read the change wrong.

On Tue, Aug 16, 2016 at 3:09 PM, John Chase <notifications@github.com>
wrote:

> In cualid/label.py
> <https://github.com/johnchase/cual-id/pull/33#discussion_r75030212>:
>
> > @@ -26,7 +26,7 @@ def get_barcodes(input,
> >                   x_start=1.9,
> >                   y_start=257.2):
> >
> > -    ids = [e.strip().split('\t')[1] for e in input]
> > +    ids = [e.strip().split('\t')[0] for e in input]
>
> This will now check the existing IDs against the second column of whatever
> file is passed. I thikn what we want to do is still check the first column,
> but when the IDs are created have the cual-id in the first column and the
> UUID in the second
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/johnchase/cual-id/pull/33/files/5d9cb7aafea8d965f1a16e20ea318ef05356b1b1#r75030212>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATEw0SGvdnwDJrsz4fv1Ij5oKURnpGt5ks5qgjUYgaJpZM4Jl2ID>
> .
>